### PR TITLE
added integration testing

### PR DESCRIPTION
**Pull request by nevil-mathew**

### CODE REVIEW

File: src/integration-test/mentor/mentor.spec.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * name : mentor.spec.js
+ * author : Nevil
+ * created-date : 28-Sep-2022
+ * Description : Integration test for mentor controllers.
+ */
+
+const { request, logIn, logError } = require('../commonTests')
+const { listMentorsSchema } = require('./responseSchema')
+const { insertMentor } = require('./mentorData')
```
Comment (rakeshSgr) [inline src/integration-test/mentor/mentor.spec.js:10]: @nevil-mathew Can we use @ insted of relative path

File: src/integration-test/mentor/mentorData.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const insertMentor = async () => {
+	try {
+		bodyData = {
+			name: 'Nevil (Mentor)',
+			email: { address: faker.internet.email(), verified: false },
+			password: faker.internet.password(),
+			isAMentor: true,
+			secretCode: process.env.MENTOR_SECRET_CODE,
+		}
+		await usersData.createUser(bodyData)
```
Comment (rakeshSgr) [inline src/integration-test/mentor/mentorData.js:13]: @nevil-mathew why you are using the local db model ?

File: src/integration-test/profile/profileData.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+const usersData = require('@db/users/queries')
```
Comment (rakeshSgr) [inline src/integration-test/profile/profileData.js:1]: @nevil-mathew why we need db model ?

File: src/integration-test/userentity/userentity.spec.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+	beforeAll(async () => {
+		userDetails = await logIn()
+		entityId = await insertEntity()
+	})
+
+	it('creating user entity', async () => {
+		let value = faker.word.adjective()
+
+		let res = await request
```
Comment (rakeshSgr) [inline src/integration-test/userentity/userentity.spec.js:24]: @nevil-mathew u can use genric method to call this

File: src/integration-test/mentor/mentorData.js (hunk 2 above)
Comment (nevil-mathew) [reply src/integration-test/mentor/mentorData.js:13]: I'm using it to seed the required data.

**Pull request merged**